code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -192,7 +192,7 @@
 	name = "spiderling"
 	desc = "It's slightly twitching in your hand. Ew..."
 	icon_state = "spiderling"
-	list_reagents = list("nutrient" = 1, "toxin" = 4)
+	list_reagents = list("nutriment" = 1, "toxin" = 4)
```

Comment:
might want to move that into a seperate PR to get it done sooner.

Reply:
Nah, I am too lazy to make separate PR for such thing.
And if you don't mind, I'll replace mobspawn to spawn strange plant in reaction for strange reagent. Because it's kinda boring and stupid.

Reply:
Yes we do mind, don't bundle unrelated changes together.